swoloszynski/kitcat
# Issue 34 create login page, #34

### What
- Use HTML5 boilerplate and materialize to create base template
- Add pages for authentication
- Add blank index page to land on after logging in
### Test
- Try to visit / without logging in, should redirect to login
- Login with wrong credentials, see error message
- Log in with valid credentials, redirect to index
- Log out, redirect to login
### Deploy notes
- On deploy, check if static files are being served correctly
